src/vma/util/sys_vars.h

```diff
@@ -669,7 +645,6 @@ extern mce_sys_var & safe_mce_sys();
 #define MCE_CQ_ADAPTIVE_MODERATION_DISABLED		(0)
```

Comment:
can't comment on line 641 - but I think MCE_MIN_RX_PREFETCH_BYTES should change to (ETH+IP+UDP)

Reply:
I don't want to change the value as its constant today for both cases but i remove the comment becuase its incorrect even in IPoIB case